java-security/src/main/java/com/sap/cloud/security/token/XsuaaToken.java

```diff
@@ -159,6 +159,19 @@ public String getSubdomain() {
 		return getAttributeFromClaimAsString(EXTERNAL_ATTRIBUTE, EXTERNAL_ATTRIBUTE_ZDN);
 	}
 
+	/**
+	 * Returns subaccount identifier which is in most cases same like the
+	 * identity zone.
+	 * DO only use this for metering purposes.
+	 * DO NOT longer use this method to get the unique tenant id! For that use {@link #getZoneId()}.
+	 *
+	 * @return subaccount identifier or {@code null}
+	 */
+	@Nullable
+	public String getSubaccountId() {
+		return getAttributeFromClaimAsString(EXTERNAL_ATTRIBUTE, EXTERNAL_ATTRIBUTE_SUBACCOUNTID);
```

Comment:
It is very similar to the implementation in `XSUserInfoAdapter` except it does not have a fallback to`ZONE_ID`: https://github.com/SAP/cloud-security-xsuaa-integration/pull/314/files#diff-47c999b39e9c3952559b7cade6014bd2R131 is this on purpose?
Can you define a test for this?

Reply:
oh, yes - thanks for that review comment!